crates/rpc/rpc/src/eth/signer.rs

```diff
@@ -26,12 +27,15 @@ pub(crate) trait EthSigner: Send + Sync {
         request: TypedTransactionRequest,
         address: &Address,
     ) -> Result<TransactionSigned>;
+
+    /// Encodes and signs the typed data according EIP-712. Payload must implement Eip712 trait.
+    fn sign_typed_data(&self, address: Address, payload: &TypedData) -> Result<Signature>;
 }
 
 /// Holds developer keys
 pub(crate) struct DevSigner {
-    addresses: Vec<Address>,
-    accounts: HashMap<Address, SecretKey>,
+    pub(crate) addresses: Vec<Address>,
+    pub(crate) accounts: HashMap<Address, SecretKey>,
```

Comment:
do we need pub crate access here?
imo should be private.

Reply:
No, no we don't. Was testing some things and committed this by mistake, made them private again.